chef/cookbooks/provisioner/templates/default/autoyast.xml.erb

```diff
@@ -278,6 +278,9 @@ sync
   <% end -%>
       <package>supportutils-plugin-susecloud</package>
 <% end -%>
+      <% @packages.each do |package| -%>
+        <package><%= package %></package>
```

Comment:
Indentation is wrong here, should be at the same level as `<package>supportutils-plugin-susecloud</package>` earlier on.